public/app.js

```diff
@@ -11,7 +11,7 @@ import route from 'can/route/';
 import Session from './models/session';
 import "can/route/pushstate/";
 import stache from "can/view/stache/";
-import "./util/prefilter";
```

Comment:
it shouldn't be imported globally because there's really no such thing.  Think about a test or a demo page that doesn't load the app view model, this helper would not be available.  

It should be imported by the template using it.  

Another example are components.  Say I needed `<bit-tabs>`.  Yes, `<bit-tabs>` is registered globally, but it should still be imported by what needs it.